benchmarks/htap/__init__.py

```diff
@@ -20,8 +20,8 @@ def add_parser(subparsers):
         'How many seconds the benchmark should run for, default: 60.'))
 
     parser.add_argument(
-        '--olap-timeout', default=900, type=int, help=(
-        'Timeout for OLAP queries in seconds, default: 900'))
```

Comment:
"in seconds" can be removed from description now

Reply:
done